src/formatDate.js

```diff
@@ -8,7 +8,33 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const fromFormatCopy = [...fromFormat];
+  const toFormatCopy = [...toFormat];
+  const deleteLastElement = fromFormatCopy.pop();
+  const partOfDate = date.split(deleteLastElement);
```

Comment:
The logic for splitting the date string using `deleteLastElement` is incorrect. This assumes that the last element of `fromFormat` is a delimiter, which may not always be the case. Consider using a more robust method to parse the date string based on the format provided.